server.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+
+// const express = require('express')
+
+const app = express();
+const port = 3025;
+
+const jason = {
+	Todos: [
+		{
+			name: "Sebastian",
+			city: "Hamburg",
+		},
+		{
+			name: "Florian",
+			city: "Frankfurt",
+		},
+	],
+};
+app.get("/", (req, res) => {
+	res.send("Hello World");
+});
+
+app.get("/todos/", (request, response) => {
+	response.json(jason); // JSON.stringify(jason)
```

Comment:
jason?

Reply:
Ach, jason ist dein Array mit den Objekten drin, nvm...

Reply:
Yes, it is just the name of the constant which contains the JSON Object

Reply:
LGTM